index.html

```diff
@@ -4973,6 +4973,14 @@ <h3>HTML Attribute State and Property Mappings</h3>
               <td class="ax"><div class="general">Use WAI-ARIA mapping</div></td>
               <td class="comments">
                 If the element includes both the `required` attribute and the `aria-required` attribute with a valid value, User Agents MUST expose only the `required` attribute value.
+                <p>
+                  If an element is <a data-cite="html/input.html#concept-input-required">required</a>, user agents MUST NOT expose the 
+                  element with an intitial invalid state (<a class="core-mapping" href="#ariaInvalidTrue">`aria-invalid="true"`</a>). 
+                  Only after a user has purposefully interacted with a required element, or attempted to submit a form and the element, or elements, do not meet
+                  <a data-cite="html/form-control-infrastructure.html#constraint-validation">constraint validation</a> would it be expected for 
+                  these required elements to be exposed in the invalid state. Until then, user agents MUST expose the elements as
+                  (<a class="core-mapping" href="#ariaInvalidFalse">`aria-invalid="false"`</a>).
```

Comment:
I find the and/or prioritization of the long sentence difficult to parse. How about adding 1), 2), like below, or break into an AND list where the LIs can encapsulate the ORs.

```suggestion
                  If an element is <a data-cite="html/input.html#concept-input-required">required</a>, user agents MUST NOT expose the 
                  element with an intitial invalid state (<a class="core-mapping" href="#ariaInvalidTrue">`aria-invalid="true"`</a>). 
                  The user agent SHOULD expose the invalid state only after <strong>1)</strong> a user has purposefully interacted with a required element, or attempted to submit a form and <strong>2)</strong> the element, or elements, do not meet
                  <a data-cite="html/form-control-infrastructure.html#constraint-validation">constraint validation</a>. Until then, user agents MUST expose the elements as
                  (<a class="core-mapping" href="#ariaInvalidFalse">`aria-invalid="false"`</a>).
```